webpack.common.js

```diff
@@ -13,13 +13,15 @@ const externals = {
 	'@wordpress/blocks': 'wp.blocks',
 	'@wordpress/editor': 'wp.editor',
 	'@wordpress/compose': 'wp.compose',
+	'@wordpress/hooks': 'wp.hooks',
 	react: 'eejs.vendor.react',
 	'react-dom': 'eejs.vendor.reactDom',
 	'react-redux': 'eejs.vendor.reactRedux',
 	redux: 'eejs.vendor.redux',
 	classnames: 'eejs.vendor.classnames',
 	lodash: 'eejs.vendor.lodash',
 	'moment-timezone': 'eejs.vendor.moment',
+	cuid: 'eejs.vendor.cuid',
```

Comment:
cuid is exposed on the `eejs.vendor` so it can be used by add-ons as well via `eejs.vendor.cuid`